tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature

```diff
@@ -315,3 +315,138 @@ Feature: Sharing files and folders with internal users
+      | share      |
+      | create     |
+      | change     |
+    Then user "User One" should be listed as "Custom role" in the collaborators list on the webUI
+    And as "user1" these resources should be listed on the webUI
+      | entry_name                  |
+      | simple-folder (2)           |
+    And these resources should be listed in the folder "simple-folder (2)" on the webUI
+      | lorem.txt |
+    When the user renames file "lorem.txt" to "new-lorem.txt" using the webUI
+    Then file "new-lorem.txt" should be listed on the webUI
+    And file "lorem.txt" should not be listed on the webUI
+    When the user creates a folder with the name "some-folder" using the webUI
+    Then folder "some-folder" should be listed on the webUI
+
+  Scenario Outline: share a file & folder with another internal user and set various custom permission
+    Given user "user2" has logged in using the webUI
+    When the user shares folder "simple-folder" with user "User One" as "Custom Role" using the webUI
+    Then user "User One" should be listed as "Viewer" in the collaborators list for folder "simple-folder" on the webUI
+    When the user sets the custom permissions for "User One" for folder "simple-folder" as:
+      | permission           |
+      | <custom-permission>  |
+    Then user "User One" should be listed as "Custom role" in the collaborators list on the webUI
+    And as "user1" these resources should be listed on the webUI
+      | entry_name                  |
+      | simple-folder (2)           |
+    And these resources should be listed in the folder "simple-folder (2)" on the webUI
+      | lorem.txt |
+    Examples:
+      | custom-permission |
+      | change            |
+      | create            |
+      | delete            |
+      | share             |
+
+  Scenario: share a file & folder with another internal user with create custom permission
+    Given user "user2" has logged in using the webUI
+    When the user shares folder "simple-folder" with user "User One" as "Custom Role" using the webUI
+    Then user "User One" should be listed as "Viewer" in the collaborators list for folder "simple-folder" on the webUI
+    When the user sets the custom permissions for "User One" for folder "simple-folder" as:
+      | permission |
+      | create     |
+    Then user "User One" should be listed as "Custom role" in the collaborators list on the webUI
+    And as "user1" these resources should be listed on the webUI
+      | entry_name                  |
+      | simple-folder (2)           |
+    And these resources should be listed in the folder "simple-folder (2)" on the webUI
+      | lorem.txt |
+    When the user creates a folder with the name "some-folder" using the webUI
+    Then folder "some-folder" should be listed on the webUI
+    Then it should not be possible to share file "lorem.txt" using the webUI
+    And it should not be possible to delete file "lorem.txt" using the webUI
+
+  @issue-1837 @skip
```

Comment:
do not skip the test but make it, so that it works not but fails when the bug is fixed see:
https://www.linkedin.com/pulse/should-you-write-acceptance-tests-bugs-artur-neumann/
and
https://doc.owncloud.com/server/developer_manual/testing/acceptance-tests.html#writing-scenarios-for-bugs